src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -106,15 +146,24 @@ public ObservableList<Person> getPersonList() {
         return persons.asUnmodifiableObservableList();
     }
 
+    @Override
+    public ObservableList<Pair> getPairList() {
+        return pairs.asUnmodifiableObservableList();
+    }
+
     @Override
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof AddressBook // instanceof handles nulls
                 && persons.equals(((AddressBook) other).persons));
+        // todo (yong jing)
+                //&& pairs.equals(((AddressBook) other).pairs));
```

Comment:
can just write `// TODO: task description`and remove the implementation

Reply:
same for the rest

Reply:
Fixed in commit 4bc2292